src/test/ui/parser/stmt_expr_attrs_placement.rs

```diff
@@ -0,0 +1,22 @@
+#![feature(stmt_expr_attributes)]
+
+// Test that various placements of the inner attribute are parsed correctly,
+// or not.
+
+fn main() {
+    let a = #![allow(warnings)] (1, 2);
+    //~^ ERROR an inner attribute is not permitted in this context
+
+    let b = (#![allow(warnings)] 1, 2);
```

Comment:
These other cases should also be emitting warnings, right?

Reply:
Well warnings are being allowed so that doesn't seem right. The point of this test is only parsing behavior tho, not the lint system.